tests/unit/test_charm.py

```diff
@@ -907,3 +909,86 @@ def test_given_magma_service_enabled_when_install_then_nothing_done(
                 ),
             ]
         )
+
+    @patch("charm.Path")
+    @patch("subprocess.run")
+    def test_when_orchestrator_available_event_then_configuration_is_installed(
+        self, patch_subprocess_run, patch_path
+    ):
+        event = Mock(
+            root_ca_certificate="root_ca_certificate_content",
+            orchestrator_address="orchestrator_address",
+            orchestrator_port="orchestrator_port",
+            bootstrapper_address="bootstrapper_address",
+            bootstrapper_port="bootstrapper_port",
+            fluentd_address="fluentd_address",
+            fluentd_port="fluentd_port",
+        )
+        self.harness.charm._on_orchestrator_available(event)
```

Comment:
How about creating an actual relation instead of this?

Reply:
Good point, not sure why I didn't think about it!

Reply:
Fixed